app/views/devise/mailer/confirmation_instructions.html.slim

```diff
@@ -11,13 +11,16 @@ table.button.expanded.large.radius
               td
                 center
                   = link_to t('mailer.confirmation_instructions.link_text'), \
-                    sign_up_create_email_confirmation_url(confirmation_token: @token), \
+                    sign_up_create_email_confirmation_url(_request_id: \
```

Comment:
Good point. I typically use `git log -S` or I look up the file in question in GitHub and look at the history to find out which commit introduced something and then I look up the PR, assuming the commit message explains the change since we've been pretty good about doing that.